blockchains/elrond/assets/WEGLD-bd4d79/info.json

```diff
@@ -0,0 +1,48 @@
+{
+    "type": "ESDT",
+    "id": "WEGLD-bd4d79",
+    "name": "WrappedEGLD",
+    "symbol": "WEGLD",
```

Comment:
```suggestion
    "symbol": "wEGLD",
```

is it `wEGLD` or `WEGLD` ?

Reply:
It's `WEGLD` - I've fixed the description.